src/styles/styles/_logo.scss

```
@@ -0,0 +1,9 @@
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+  &-image {
```

```suggestion
  height: 40px;
  width: 40px;
  
  &-image {
```
It would be great to create a variable for logo size